packages/nextjs-template/hooks/useIFrameHeightAdjusterr.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
nit: there's an extra 'r' in the file name: 'Adjusterr'

Reply:
Fixed